website/docs/d/aaep_to_domain.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+subcategory: "Access Policies"
+layout: "aci"
+page_title: "ACI: aci_aaep_to_domain"
+sidebar_current: "docs-aci-data-aaep-to-domain"
+description: |-
+  Data source for ACI Attachable Access Entity Profile - VMM, Physical or External domain interfaces.
+---
+
+# aci_aaep_to_domain #
+
+Data source for ACI Attachable Access Entity Profile - VMM, Physical or External domain interfaces.
```

Comment:
Data source for ACI Attachable Access Entity Profile (AAEP) to domain (VMM, Physical or External domain) relationships.